### DISCUSSION TITLE
Jaeger in CronJob

### DISCUSSION THREAD

**Discussion by Kasheeew** [jaegertracing/jaeger#2942] [Q&A]
Okay so i got a CronJob  which is using OpenTracing. But where do i put the annotations into the yaml? Nothing worked for me!

![Screenshot_1](https://user-images.githubusercontent.com/75539945/115109089-1d357f00-9f74-11eb-9cbf-2662ea3aad81.png)
I know how it should look like in a Deployment:
![grafik](https://user-images.githubusercontent.com/75539945/115109167-743b5400-9f74-11eb-8173-bc1245d9587a.png)

          
If something is unclear let me know! Thanks for any kind of help!

**Comment by jpkrohling** [comment]
The Jaeger Operator will not add sidecars to cronjobs, only to deployments. In fact, sidecars are not well-supported in Kubernetes, as they would keep the pod up as running even when the main container has finished the job.
